V2/time_tracker/time_entries/_application/_time_entries/_get_latest_entries.py

```diff
@@ -0,0 +1,47 @@
+import json
+
+import azure.functions as func
+
+from ... import _domain
+from ... import _infrastructure
+from time_tracker._infrastructure import DB
+
+
+def get_latest_entries(req: func.HttpRequest) -> func.HttpResponse:
+    database = DB()
+    time_entry_dao = _infrastructure.TimeEntriesSQLDao(database)
+    time_entry_service = _domain.TimeEntryService(time_entry_dao)
+    use_case = _domain._use_cases.GetLastestTimeEntryUseCase(time_entry_service)
+
+    try:
+        owner_id = req.params.get("owner_id")
+        limit = req.params.get("limit")
+
+        if not owner_id:
+            return func.HttpResponse(
+                body="No owner id found",
+                status_code=404,
+                mimetype="application/json"
```

Comment:
We could create a constants file for server responses and errors instead of hardcoding the strings

Reply:
We are going to create a ticket, this or next week to make these changes and standardize the code to avoid these possible errors when writing code, so I do not make the change in this pr